src/components/views/rooms/ReadReceiptMarker.tsx

```diff
@@ -16,61 +16,74 @@ limitations under the License.
 */
 
 import React, { createRef } from 'react';
-import PropTypes from 'prop-types';
+import { RoomMember } from 'matrix-js-sdk/src';
```

Comment:
```suggestion
import { RoomMember } from "matrix-js-sdk/src/models/room-member";
```

I do have a feeling that not doing this has caused problems before but I might be wrong

(same in other files)